data_juicer/ops/mapper/extract_qa_mapper.py

```diff
@@ -80,14 +122,21 @@ def _extract_qa(self, output):
     def process(self, sample, rank=None):
         model, processor = get_model(self.model_key, rank, self.use_cuda())
 
-        inputs = processor(sample[self.text_key],
-                           return_tensors='pt').to(model.device)
-        response = model.generate(**inputs)
-        output = processor.decode(response.cpu()[0], skip_special_tokens=True)
+        if self.enable_vllm:
+            response = model.generate([sample[self.text_key]],
+                                      self.sampling_params)
+            output = response[0].outputs[0].text
+        else:
```

Comment:
We can leave a TODO item: we may extend these LLM-based OPs into API-based implementation in future PRs.